docs/efit_data.ipynb

```diff
@@ -0,0 +1,8891 @@
+       "      <td>30470</td>\n",
+       "      <td>efm</td>\n",
+       "      <td>s3://mast/level1/shots/30470.zarr/efm</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>99066</th>\n",
+       "      <td>Basic EFIT</td>\n",
+       "      <td>Not Checked</td>\n",
+       "      <td>1b50bfb9-e9f3-5c23-8f5b-5821e441d1b7</td>\n",
+       "      <td>30471</td>\n",
+       "      <td>efm</td>\n",
+       "      <td>s3://mast/level1/shots/30471.zarr/efm</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>13621 rows × 6 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      description      quality                                  uuid  shot_id  \\\n",
+       "1      Basic EFIT  Not Checked  fd0a0dc4-1ed8-546f-8c02-455061374fd4    11695   \n",
+       "3      Basic EFIT  Not Checked  c553a338-249f-58d5-9ba8-14f5c507c4e4    11696   \n",
+       "5      Basic EFIT  Not Checked  f2ce95e9-bfe8-595d-a1eb-d860de377ab2    11697   \n",
+       "7      Basic EFIT  Not Checked  f1fbd4b8-d68b-54bd-a9c0-209fddf9cef0    11698   \n",
+       "9      Basic EFIT  Not Checked  798d44ea-0773-5020-8c3c-083b1bdb3aab    11699   \n",
+       "...           ...          ...                                   ...      ...   \n",
+       "99006  Basic EFIT  Not Checked  72422662-d3dd-5bb9-abdc-5e5e3427587e    30467   \n",
+       "99021  Basic EFIT  Not Checked  5f073f88-7f09-53ca-b204-e9c620b0672d    30468   \n",
+       "99036  Basic EFIT  Not Checked  0bcd11d2-f8f5-5ca3-94e6-235c13b1434d    30469   \n",
+       "99051  Basic EFIT  Not Checked  91cbcfda-7434-55be-a760-80493d6e8f46    30470   \n",
+       "99066  Basic EFIT  Not Checked  1b50bfb9-e9f3-5c23-8f5b-5821e441d1b7    30471   \n",
+       "\n",
+       "      name                                    url  \n",
+       "1      efm  s3://mast/level1/shots/11695.zarr/efm  \n",
+       "3      efm  s3://mast/level1/shots/11696.zarr/efm  \n",
+       "5      efm  s3://mast/level1/shots/11697.zarr/efm  \n",
+       "7      efm  s3://mast/level1/shots/11698.zarr/efm  \n",
+       "9      efm  s3://mast/level1/shots/11699.zarr/efm  \n",
+       "...    ...                                    ...  \n",
+       "99006  efm  s3://mast/level1/shots/30467.zarr/efm  \n",
+       "99021  efm  s3://mast/level1/shots/30468.zarr/efm  \n",
+       "99036  efm  s3://mast/level1/shots/30469.zarr/efm  \n",
+       "99051  efm  s3://mast/level1/shots/30470.zarr/efm  \n",
+       "99066  efm  s3://mast/level1/shots/30471.zarr/efm  \n",
+       "\n",
+       "[13621 rows x 6 columns]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "catalog  = intake.open_catalog('https://mastapp.site/intake/catalog.yml')\n",
+    "sources_df = catalog.index.level1.sources().read()\n",
+    "\n",
+    "# Can also use name == 'aye' for thompson scattering edge data.\n",
```

Comment:
Since you have a TS example, this comment can probably go? It seems out of place for an equilibrium example.